app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -284,6 +289,53 @@ def transform_metrics(img_metrics)
           new_metrics
         end
 
+        # Generate a hash for image references information that can be linked to Alert
+        # @return A hash with region_id => {:key : 'What region', :side: 'Front|Back'}
+        def parse_document_region
+          region_details = {}
+          image_sides = {}
+          true_id_product[:ParameterDetails].each do |detail|
+            next unless detail[:Group] == 'DOCUMENT_REGION' ||
+                        (detail[:Group] == 'IMAGE_METRICS_RESULT' &&
+                          %w[ImageMetrics_Id  Side].include?(detail[:Name]))
+            inner_val = detail.dig(:Values).collect { |value| value.dig(:Value) }
```

Comment:
same about dig on a top level item. Also I'd use `.map` for consistency instead of `.collect`
```suggestion
            inner_val = detail[:Values].map { |value| value[:Value] }
```

Reply:
fixed.